subprojects/plugins/src/main/java/org/gradle/api/plugins/internal/JvmPluginsHelper.java

```diff
@@ -171,43 +170,6 @@ public static void configureJavaDocTask(@Nullable String featureName, SourceSet
         }
     }
 
-    /**
-     * @deprecated Use {@link #configureDocumentationVariantWithArtifact(String, String, String, List, String, Object, AdhocComponentWithVariants, ConfigurationContainer, TaskContainer, ObjectFactory, FileResolver)}
-     * instead. Passing {@code null} for the FileResolver will not be legal after this is removed, please provide one.
-     */
-    @Deprecated
-    public static void configureDocumentationVariantWithArtifact(
```

Comment:
There is another method with the same name in `DefaultJvmVariantBuilder`, and it looks quite similar. Do you know if these can be merged?

Reply:
These do look like they would produce identical variants, at least at first glance after following what attributes they would create.  I'm going to explore merging them in a separate PR.  Good catch!